parm/config/gfs/config.resources

```diff
@@ -209,8 +209,8 @@ case ${step} in
   # The wavepost*pnt* jobs are I/O heavy and do not scale well to large nodes.
   # Limit the number of tasks/node to 40.
   "wavepostbndpnt")
```

Comment:
Sorry for the late comment on this, but should these resources also be adjusted for GEFS?

Reply:
@JessicaMeixner-NOAA Do the GEFS resources also need to be updated?

Reply:
These point jobs are only used in GFS, but I'll defer to @sbanihash  if she would like these updated in GEFS or not.

Reply:
Thanks!